salt/transport/ipc.py

```diff
@@ -68,6 +73,24 @@ def start(self):
         )
         self._started = True
 
+    def publish(self, msg):
+        '''
+        Publish a `msg` to all connected clients
+        '''
+        payload = salt.transport.frame.frame_msg(msg, raw_body=True)
+        to_remove = []
+        for client in self.clients:
+            try:
+                # Don't wait on the future, best effort only
+                f = client.write(payload)
+                self.io_loop.add_future(f, lambda f: True)
```

Comment:
I'm sure this is a Tornado-ism that I don't grok, but why is it necessary to add the future to the loop with this callback? (This isn't a problem, I'm just curious.)